lnwallet/interface_test.go

```diff
@@ -205,7 +205,7 @@ func assertTxInWallet(t *testing.T, w *lnwallet.LightningWallet,
 	// We'll fetch all of our transaction and go through each one until
 	// finding the expected transaction with its expected confirmation
 	// status.
-	txs, err := w.ListTransactionDetails()
+	txs, err := w.ListTransactionDetails(0, btcwallet.UnconfirmedHeight)
```

Comment:
It would be nice to have tests other than the full interval.

Reply:
it would indeed 🙈

Reply:
Do we need to test that lnd side? Reversal of order is handled by `GetTransactions` in btcwallet, so it should be covered there?